networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -30,3 +71,4 @@
   - name: Verify tuned stac profile on producer and consumer nodes
     shell: tuned-adm verify
     ignore_errors: yes
```

Comment:
I forget why you are ignoring errors here?

Reply:
not any specific reason, actually we want to know if it fails to to make sure we have the right tuned profile running